hoprd/rest-api/src/network.rs

```diff
@@ -48,3 +48,40 @@ pub(super) async fn price(State(state): State<Arc<InternalState>>) -> impl IntoR
         Err(e) => (StatusCode::UNPROCESSABLE_ENTITY, ApiErrorStatus::from(e)).into_response(),
     }
 }
+
+#[derive(Debug, Clone, serde::Serialize, utoipa::ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct TicketProbabilityResponse {
+    /// Winning probability of a ticket.
+    probability: String,
```

Comment:
Why is this a string? Probability can only be `<0,1>`. Ever. The same is on the other side, let's make this a `f64`.

Reply:
Agreed. I defined it as a `String` to match the `TicketPriceResponse` struct, which define the price as a `String` too. 
Changing for the ticket probability